doc/active_docs/account_management_api.json

```diff
@@ -405,7 +405,6 @@
             "in": "path",
             "description": "ID of the application.",
             "x-data-threescale-name": "application_ids",
-            "required": true,
```

Comment:
That's not the right param. `id` is still required (it cannot NOT be required, as it's a path parameter). The one that should stop being required is `name`, as the Jira says.

Reply:
Okay got it.

Reply:
done